index.js

```diff
@@ -41,6 +42,11 @@ server.express.post('/schedule-daily-mails', scheduleDailyMails);
 server.express.post('/update-intercom', updateIntercom);
 
 server.express.post('/posthook-receiver', bodyParser.json(), posthookReceiver);
+server.express.post(
+	'/lifetime-payment',
+	bodyParser.raw({type: 'application/json'}),
```

Comment:
why not using `bodyParser.json` as above? Plus, no need to parse in the callback.